tensorflow/lite/micro/micro_profiler.h

```diff
@@ -16,15 +16,28 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_MICRO_MICRO_PROFILER_H_
 #define TENSORFLOW_LITE_MICRO_MICRO_PROFILER_H_
 
+#include <cinttypes>
+#include <cstdint>
+#include <cstring>
+
+#include "tensorflow/lite/kernels/internal/compatibility.h"
 #include "tensorflow/lite/micro/compatibility.h"
+#include "tensorflow/lite/micro/micro_log.h"
 #include "tensorflow/lite/micro/micro_profiler_interface.h"
+#include "tensorflow/lite/micro/micro_time.h"
 
 namespace tflite {
 
+enum class MicroProfilerLogFormat {
+  HumanReadable,
+  Csv,
+};
+
 // MicroProfiler creates a common way to gain fine-grained insight into runtime
-// performance. Bottleck operators can be identified along with slow code
+// performance. Bottleneck operators can be identified along with slow code
 // sections. This can be used in conjunction with running the relevant micro
 // benchmark to evaluate end-to-end performance.
+template <int MAX_EVENTS>
```

Comment:
This should be `template <int MAX_EVENTS = 4096>` so that all existing applications using the profiler do not need to be changed.

Reply:
Done